src/repl.jl

```diff
@@ -133,6 +136,18 @@ function hideprompt(f)
   if INIT_COMPLETE[]
     # Escape REPL modes, then write sentinel and trigger chars.
     can_write_to_terminal = something(Atom.@rpc(writeToTerminal("\b\a$(REPL_SENTINEL_CHAR)$(REPL_TRIGGER_CHAR)")), true)
```

Comment:
Do we still need this line ? Can't we always `blockinput` for this pass ?

Reply:
Ah, good point -- we should only start the timer if `can_write_to_terminal == true`.